test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1363,11 +1363,11 @@ var Annotations = map[string]string{
 
 	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router reports the expected host names in admitted routes' statuses": " [Suite:openshift/conformance/parallel]",
 
-	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router should override the route host for overridden domains with a custom value [apigroup:image.openshift.io]": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
+	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router should override the route host for overridden domains with a custom value": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
 
 	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router should override the route host with a custom value": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
 
-	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router should run even if it has no access to update status [apigroup:image.openshift.io]": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
+	"[sig-network][Feature:Router][apigroup:route.openshift.io] The HAProxy router should run even if it has no access to update status": " [Skipped:Disconnected] [Suite:openshift/conformance/parallel]",
```

Comment:
Any test name changes need to be mapped into the ci-test-mapping repo so we maintain historical continuity.

These tests are owned by this component: https://github.com/openshift-eng/ci-test-mapping/blob/main/pkg/components/networking/router/component.go

Here's an example of how to add the rename map https://github.com/openshift-eng/ci-test-mapping/pull/39/files#diff-b36e285c7d8fba736f1c50f6688e480c4865bbe8ed67b808b440a3dfbade61d5

Reply:
Yes, I was part of the renaming last year with apigroup tagging, thanks!